web/vue.config.js

```diff
@@ -8,4 +8,7 @@ module.exports = {
   transpileDependencies: [
     'vuetify',
   ],
+  devServer: {
+    port: 8085,
+  }
```

Comment:
this is beyond "docs"... was it intended to be committed and if so - better goes into separate commit with a reason what for

Reply:
Previously, the docs instructed developers to always run `yarn run serve --port 8085`. This internalizes that configuration, which makes the developer experience simpler and more consistent.

Reply:
To further clarify, the only impact of this change is to simplify the commands typed by front-end developers. Indeed, for developers that really want to run the dev server on another port, the `--port` option will continue to override this.

It does not effect the way that the site is built for production.

Reply:
thanks!  I just wanted to make sure that nothing "leaked" since title (or PR and the commit where the change is mixed in) said "Improve ... docs". Such descriptions better be placed into the commit messages introducing the changes, that could help later to investigate how the change came about if comes needed.